serverlessworkflow/modules/ROOT/pages/cloud/operator/add-custom-ca-to-a-workflow-pod.adoc

```diff
@@ -0,0 +1,188 @@
+The example mainly uses the standard http client so alternatively we could mount the cacerts to a different location and
+configure the Java runtime to load the new keystore with a system property `-Djavax.net.ssl.trustStore`.
+Note that libraries like resteasy don't respect that flag and may need to programmatically set the trust store location.
+
+```yaml
+ containers:
+   - command:
+     - /bin/bash
+     - -c
+     - |
+       curl -L https://gist.githubusercontent.com/rgolangh/b949d8617709d10ba6c690863e52f259/raw/bdea4d757a05b75935bbb57f3f05635f13927b34/Main.java -o curl.java
+       java curl.java https://kubernetes
+     image: registry.access.redhat.com/ubi9/openjdk-17
+     imagePullPolicy: Always
+     name: openjdk-17
+     volumeMounts:
+       - mountPath: /lib/jvm/java-17/lib/security
+         name: new-cacerts
+         readOnly: true
+       - mountPath: /var/run/secrets/kubernetes.io/serviceaccount
+         name: kube-api-access-5npmd
+         readOnly: true
+```
+
+Notice the volume mount of the previously mutated keystore.
+
+
+=== Full working example
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: root-ca-to-cacerts
+spec:
+  initContainers:
+    - name:  add-kube-root-ca-to-cacerts
+      image: registry.access.redhat.com/ubi9/openjdk-17
+      volumeMounts:
+        - mountPath: /opt/new-cacerts
+          name: new-cacerts 
+      command:
+        - /bin/bash
+        - -c
+        - |
+          cp $JAVA_HOME/lib/security/cacerts /opt/new-cacerts/
+          chmod +w /opt/new-cacerts/cacerts
+          keytool -importcert -no-prompt -keystore /opt/new-cacerts/cacerts -storepass changeit -file /var/run/secrets/kubernetes.io/serviceaccount/ca.crt 
+  containers:
+    - command:
+      - /bin/bash
+      - -c
+      - |
+        curl -L https://gist.githubusercontent.com/rgolangh/b949d8617709d10ba6c690863e52f259/raw/bdea4d757a05b75935bbb57f3f05635f13927b34/Main.java -o curl.java
+        java curl.java https://kubernetes
+      image: registry.access.redhat.com/ubi9/openjdk-17
+      imagePullPolicy: Always
+      name: openjdk-17
+      volumeMounts:
+      - mountPath: /lib/jvm/java-17/lib/security/
+        name: new-cacerts
+        readOnly: true
+      - mountPath: /var/run/secrets/kubernetes.io/serviceaccount
+        name: kube-api-access-5npmd
+        readOnly: true
+  volumes:
+  - name: new-cacerts
+    emptyDir: {}
+  - name: kube-api-access-5npmd
+    projected:
+      sources:
+      - serviceAccountToken:
+          path: token
+      - configMap:
+          items:
+          - key: ca.crt
+            path: ca.crt
+          name: kube-root-ca.crt
+```
+
+=== Serverless Workflow Example
+
+Similar to a deployment spec, a serverless workflow has a spec.podTemplate, with minor differences, but the change is almost identical.
+In this case we are mounting some ingress ca bundle because we want our workflow to reach the `.apps.my-cluster-name.my-cluster-domain` SSL endpoint.
```

Comment:
```suggestion
In this case we are mounting some ingress CA bundle because we want our workflow to reach the `.apps.my-cluster-name.my-cluster-domain` SSL endpoint.
```